exist-core/src/main/java/org/exist/xslt/TransformerFactoryAllocator.java

```diff
@@ -22,7 +22,7 @@
 package org.exist.xslt;
 
 import java.util.Hashtable;
-import java.util.Iterator;
+import java.util.Map;
```

Comment:
Have you looked at possible memory or performance implications for this change?

Reply:
I suggested iterating over the map entry set instead of the keys and the access the value for each key in a separate operation. I do not actually understand your question here...

Reply:
As @reinhapa it saves a few lookups, as now it is iterated directly over all items.

Reply:
@reinhapa I want to understand what I am looking at so that I can do it like this in the future.